packages/suite-desktop-ui/src/support/DesktopUpdater/JustUpdated.tsx

```diff
@@ -40,11 +40,7 @@ export const JustUpdated = ({ onCancel }: AvailableProps) => {
                 </>
             }
         >
-            <Card
-                maxHeight={400}
```

Comment:
We should use `maxHeight` for `Modal`, not for `Card`, but we don't have this prop yet. This caused visual glitch while scrolling.